static/templates/settings/organization_permissions_admin.hbs

```diff
@@ -3,8 +3,10 @@
 
         <div id="org-join-settings" class="settings-subsection-parent">
             <div class="subsection-header">
-                <h3>{{t "Joining the organization" }}</h3>
-                <i class="fa fa-info-circle settings-info-icon realm_message_retention_tooltip tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="{{t 'Only owners can change these settings.' }}"></i>
+                <h3>
+                    {{t "Joining the organization" }}
+                    <i class="fa fa-info-circle settings-info-icon realm_message_retention_tooltip tippy-zulip-tooltip" aria-hidden="true" data-tippy-content="{{t 'Only owners can change these settings.' }}"></i>
```

Comment:
While we are at this, we can remove the `realm_message_retention_tooltip` class, since that class is not used for any CSS or something else, and also this heading is not for the message retention setting. This change can be done in a separate commit.